fontir/src/ir.rs

```diff
@@ -552,6 +559,9 @@ impl GlobalMetrics {
         *self.values_mut(metric).entry(pos).or_insert(value.into())
     }
 
+    /// Specify the value of a metric at a specific location, if we have one;
+    /// this is a helper build method to reduce boilerplate. If there is an
+    /// existing value, it will be overwritten.
```

Comment:
```suggestion
    /// A helper method for setting possibly missing values.
    ///
    ///
    /// This is a helper build method to reduce boilerplate. If there is an
    /// existing value, it will be overwritten.
```

Reply:
Trimmed extra line locally and applied